config/initializers/figaro.rb

```diff
@@ -1,6 +1,6 @@
 require Rails.root.join('lib', 'config_validator.rb')
 
-Figaro.require_keys(
+Figaro.require_keys([
```

Comment:
I vote to keep the splat/`*args`

Reply:
To me it's a list of keys we want to require, but I don't feel super strongly